src/chemcoord/_cartesian_coordinates/xyz_functions.py

```diff
@@ -84,6 +85,104 @@ def open_file(i: int) -> None:
         raise ValueError("Argument is neither list nor Cartesian.")
 
 
+@overload
+def to_multiple_xyz(
+    cartesian_list: Iterable[Cartesian],
+    buf: None = None,
+    sort_index: bool = ...,
+    overwrite: bool = ...,
+    float_format: Callable[[float], str] = ...,
+) -> str: ...
+
+
+@overload
+def to_multiple_xyz(
+    cartesian_list: Iterable[Cartesian],
+    buf: PathLike,
+    sort_index: bool = ...,
+    overwrite: bool = ...,
+    float_format: Callable[[float], str] = ...,
+) -> None: ...
+
+
+def to_multiple_xyz(
+    cartesian_list: Iterable[Cartesian],
+    buf: PathLike | None = None,
+    sort_index: bool = True,
+    overwrite: bool = True,
+    float_format: Callable[[float], str] = "{:.6f}".format,
+) -> str | None:
+    """Write a list of Cartesians into an xyz file.
+
+    .. note:: Since it permamently writes a file, this function
+        is strictly speaking **not sideeffect free**.
+        The list to be written is of course not changed.
+
+    Args:
+        cartesian_list :
+        buf : StringIO-like, optional buffer to write to
+        sort_index : If sort_index is true, the Cartesian
+            is sorted by the index before writing.
+        overwrite : May overwrite existing files.
+        float_format (one-parameter function): Formatter function
+            to apply to column’s elements if they are floats.
+            The result of this function must be a unicode string.
+
+    Returns:
+        output : string (or unicode, depending on data and options)
+    """
```

Comment:
I would altogether remove the 
```python3
    Returns:
        output : string (or unicode, depending on data and options)
```

It is not correct anymore and the correct info is picked up from the type-hints.